src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  font-family: "Roboto", serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding: 10px 0 10px 50px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav-list {
+  margin-top: 19px;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav-item {
+  padding-left: 20px;
+}
+
+.nav-link {
+  padding: 23px 0 18.5px;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.nav-link:hover {
+  color: var(--blue);
+}
+
+.is-active {
+  color: var(--blue);
+}
+
+.is-active:active:after {
```

Comment:
```suggestion
.is-active::after {
```
<img width="168" alt="image" src="https://user-images.githubusercontent.com/66749919/213849344-eb2ded1a-4d72-4b06-b647-803cdb6f382b.png">